docroot/themes/custom/uids_base/scss/admin.scss

```diff
@@ -766,6 +766,14 @@ body.toolbar-vertical.toolbar-tray-open {
   }
 }
 
+// Hide certain system blocks.
+.layout-builder-block--placeholder {
+  &.block-extra-field-blocknodepagecontent-moderation-control,
+  &.block-extra-field-blocknodepagelinks {
+    @include element-invisible;
+  }
+}
```

Comment:
I don't understand why this is here.

Reply:
Okay, I'm assuming that this is related to #2078. I don't think it's a good idea to remove the display of these items. For better or worse, if they are placed via LB, I think people should be able to see that they are there. I think it would be better idea to remove "Links" from the template and possibly remove it as an option for placement. For moderation control, I think that we should put it in its own one column section at the very beginning of the template and lock that section so that people can't move it.

Reply:
I will remove that.